src/ostree/ot-admin-builtin-deploy.c

```diff
@@ -207,8 +207,12 @@ ot_admin_builtin_deploy (int argc, char **argv, OstreeCommandInvocation *invocat
 
   if (opt_kernel_argv_delete)
```

Comment:
Minor nit; this conditional can now be dropped because it's redundant with the `for` loop.